content/05-appendix.md

```diff
@@ -0,0 +1,34 @@
+# Appendix
+## Managing work vs personal personas
```

Comment:
I think the word 'personas' is probably not the best choice for this. In my opinion, it's about managing different "sub-identities" in Git. Or more explicitly, simply say, "managing work vs personal email in git".

For me, different personas would mean different usernames/names (or in GitHub terms, GitHub id).